components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/config/OAuthServerConfiguration.java

```diff
@@ -2068,6 +2077,18 @@ private void parseOpenIDConnectConfig(OMElement oauthConfigElem) {
         }
     }
 
+    private void parseHashAlgorithm(OMElement oauthConfigElem) {
+
+        OMElement hashingAlgorithmElement = oauthConfigElem
+                .getFirstChildWithName(getQNameWithIdentityNS(ConfigElements.HASH_ALGORITHM));
+        if (hashingAlgorithmElement != null) {
+            hashAlgorithm = hashingAlgorithmElement.getText();
+        }
+        if (log.isDebugEnabled()) {
+            log.debug("Hash algorithm was set to : " + hashAlgorithm);
```

Comment:
Debug log should move to above if block. Else it's default value

Reply:
In the begining of the class the hashAlgorithm variable is set to a default value[1].
Therefore it will always print a value 

https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/807/files#diff-fb1b0d809f84709ea4e8a248eddbc573R211